src/persistCache.ts

```diff
@@ -0,0 +1,21 @@
+import CachePersistor from './CachePersistor';
+import { ApolloPersistOptions } from './types';
+
+export default <T>(options: ApolloPersistOptions<T>) => {
+  if (!options.cache) {
+    throw new Error(`
+      In order to persist your Apollo Cache, you need to pass in a cache.
+      Please see https://www.apollographql.com/docs/react/basics/caching.html for our default InMemoryCache.
+    `);
+  }
+
+  if (!options.storage) {
+    throw new Error(`
+      In order to persist your Apollo Cache, you need to pass in an underlying storage provider.
+      Please see https://github.com/apollographql/apollo-cache-persist#storage-providers
+    `);
+  }
```

Comment:
These warnings are great for DX!

Two thoughts:

1. Do you mind moving them to the `CachePersistor` constructor, since advanced users could also benefit from these errors?
1. Can you replace the template string with a manually concatenated string? All of the extra whitespace in these messages looks really funky in the YellowBox on React Native (due to lots of awkward wrapping).

Reply:
Good call! I can do both of those things :)